onyo/onyo_arguments.py

```diff
@@ -10,36 +10,49 @@
         required=False,
         default=Path.cwd(),
         type=directory,
-        help='Run Onyo commands from inside of DIR'),
+        help="""
+            Run as if Onyo is started in DIR instead of the current working
+            directory.
+        """
+    ),
 
     'debug': dict(
         args=('-d', '--debug'),
         required=False,
         default=False,
         action='store_true',
-        help='Enable debug logging'),
+        help="""
+            Enable debug logging.
+        """
+    ),
 
     'version': dict(
         args=('-v', '--version'),
         action='version',
         version='%(prog)s {version}'.format(version=__version__),
-        help="Print onyo's version and exit"),
+        help="""
+            Print Onyo's version and exit.
+        """
+    ),
 
     'quiet': dict(
         args=('-q', '--quiet'),
         required=False,
         default=False,
         action='store_true',
-        help=(
-            'Silence messages printed to stdout. Does not suppress interactive '
-            'editors. Requires the --yes flag')),
+        help="""
+            Silence messages printed to stdout; does not suppress interactive
```

Comment:
Again, nitpick, but I notice that you use one time a `;` and another time a `.` and I am not totally sure what motivates different sentence dividers.

Reply:
It's a stylistic choice. This is something that there are no clear rules on because we're dealing with sentence fragments. In this case, I chose to use a semi-colon because it was connected to the first idea. The period was then used to begin a new idea.

If you find this to be confusing (vs a nitpick) let me know and I'll commit a more clear approach.

Let me know.

Reply:
It is definitely not confusing and can stay in this form without any loss.
I just felt like it interrupted the flow of reading for like 20 millisecs, hence "nitpick" felt really the only word describing it ;)

Reply:
Cool; thanks for the feedback!

Leaving this as is and resolving.